test/unit/http_method/Makefile

```
@@ -49,17 +50,35 @@ MKDIR		:=	mkdir -p
 all: $(NAME)
 
 $(NAME): $(OBJS)
+	@make init
 	$(CXX) -o $@ $^
 
+# create test file and directory
+.PHONY	: init
+init: $(TEST_CASE_DIR)/no_authority_file $(TEST_CASE_DIR)/no_authority_directory
+
+$(TEST_CASE_DIR)/no_authority_file:
+	-(mkdir -p $(TEST_CASE_DIR) && touch $@ && chmod 000 $@)
+
+$(TEST_CASE_DIR)/no_authority_directory:
+	-(mkdir -p $(TEST_CASE_DIR) && mkdir -p $@ && chmod 000 $@)
```

この二つのコマンドは()なくても動きそう
```suggestion
$(TEST_CASE_DIR)/no_authority_file:
	-mkdir -p $(TEST_CASE_DIR) && touch $@ && chmod 000 $@

$(TEST_CASE_DIR)/no_authority_directory:
	-mkdir -p $(TEST_CASE_DIR) && mkdir -p $@ && chmod 000 $@
```